### DISCUSSION TITLE
Derive Public Key from a Private Key

### DISCUSSION THREAD

**Discussion by bryanvaz** [panva/jose#177] [Q&A]
Quick question: is there an existing function in `KeyLike` to rederive a public key from a private key?

**Comment by panva** [comment]
No. In node you can use crypto.createPublicKey, in the browser there's no api for such.

**Reply by bryanvaz** [reply]
LOL. Ok time for some hacky coding. Will submit a PR if I get it working.

**Reply by treeder** [reply]
@panva so this library doesn't provide a function for it?
